### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by mdellabitta**
A few basic unit tests for the utility js we wrote.

Run by doing `node --test`.

### CODE REVIEW

**Review by bensteinberg — APPROVED** [review]
👍

File: src/lib/isEmpty.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 	} else if (Array.isArray(obj) || typeof obj === "string") {
 		return obj.length === 0;
 	} else if (
 		["boolean", "number", "bigint", "symbol", "function"].includes(typeof obj)
 	) {
-		throw new Error(f`Unsuported type: ${typeof obj}`);
+		throw new Error(`Unsuported type: ${typeof obj}`);
 	} else {
 		return Object.keys(obj).length === 0;
 	}
```
Comment (rebeccacremona) [inline src/lib/isEmpty.js:12]: Oh dear, I am such a python programmer.....

Comment (bensteinberg) [reply src/lib/isEmpty.js:12]: Oh, and I missed a typo!

Comment (mdellabitta) [reply src/lib/isEmpty.js:12]: Crap, I'll put up another PR!

**Pull request merged**